tests/api/util_test.go

```diff
@@ -0,0 +1,87 @@
+package api
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/golang-migrate/migrate/v4"
+	_ "github.com/golang-migrate/migrate/v4/database/postgres" // postgres driver for golang_migrate
+	_ "github.com/golang-migrate/migrate/v4/source/file"       // support file scheme for golang_migrate
+	_ "github.com/golang-migrate/migrate/v4/source/github"     // support github scheme for golang_migrate
+	oasisConfig "github.com/oasisprotocol/oasis-sdk/client-sdk/go/config"
+	connection "github.com/oasisprotocol/oasis-sdk/client-sdk/go/connection"
+	"github.com/stretchr/testify/require"
+
+	"github.com/oasisprotocol/oasis-indexer/analyzer/consensus"
+	"github.com/oasisprotocol/oasis-indexer/cmd/common"
+	"github.com/oasisprotocol/oasis-indexer/config"
+	"github.com/oasisprotocol/oasis-indexer/log"
+	source "github.com/oasisprotocol/oasis-indexer/storage/oasis"
+	fileSource "github.com/oasisprotocol/oasis-indexer/storage/oasis/nodeapi/file"
+)
+
+const ConfigFile = "/Users/andrewlow/oasis/oasis-block-indexer/tests/api/api-dev.yml"
+const DestFile = "/Users/andrewlow/oasis/oasis-block-indexer/tests/api/data/consensus"
+
+func TestDumpConsensusNodeData(t *testing.T) {
```

Comment:
This runs the analyzer so as to populate the cache, right?

I would much prefer for this to be expressible through the config, so we can run with pogreb caching enabled even in prod. (And we'd manually run it once in a blue moon to generate the node responses for, say, ~1000 blocks so that we have inputs for e2e tests.)
I have hopes for this caching beyond just tests :)

Reply:
Yup, I'm using this as a placeholder for now until I make changes to the config. I just didn't want to create unnecessary conflcts with #352 . I think I'll eventually be able to remove this file entirely and use makefile commands + two diff config files instead.